### PR TITLE
Add maxHeight to Cover Image so it doesn't look bad on web

### PR DESCRIPTION
**Pull request by ashwin-narkar**
| Before | After |
| --- | --- |
| image | image |

### CODE REVIEW

File: app/[event]/event.tsx [hunk 1]
```diff
@@ -595,10 +595,11 @@ const styles = {
         marginBottom: '1rem',
     },
     coverPhoto: {
-        width: '100%',
-        height: 'auto',
+        maxWidth: '100%',
+        maxHeight: '600px',
         margin: '1rem 0',
         borderRadius: '4px',
+        resizeMode: 'center',
```
Comment (ChaseC99) [inline app/[event]/event.tsx]: does resizeMode do anything?

preview lgtm but i don't see resizeMode in the final css

Comment (ashwin-narkar) [reply app/[event]/event.tsx]: oh good catch, it should determine the stretching of the image if its smaller than the container. center will make sure the image never gets stretched. It looks like the style was deprecated in lieu of a HTML attr though

Comment (ashwin-narkar) [reply app/[event]/event.tsx]: https://www.w3schools.com/jsref/prop_style_objectfit.asp

Used object fit instead

**Review by ChaseC99 — APPROVED** [review]
So much shareholder value!!

**Pull request merged**